hikari/impl/bot.py

```diff
@@ -941,7 +939,7 @@ async def _set_close_flag(self, signame: str, signum: int) -> None:
         # solution.
         _LOGGER.debug("received interrupt %s (%s), will start shutting down shortly", signame, signum)
 
-        await self.close(force=False)
+        await self.close()
```

Comment:
Does this have to call self.close or can this either just call self._closing_event.set() or self._close()?